apps/userdashboard/views.py

```diff
@@ -46,9 +45,6 @@ def projects(self):
         projects = Project.objects.filter(
             follow__creator=self.request.user, follow__enabled=True
         )
-        for project in projects:
-            project.latest_comments_num = helpers.get_num_latest_comments(project)
```

Comment:
I couldn't  find any place we use it, but it makes the site load very slow

Reply:
You are right, we don't use it!